hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/web/ozShell/bucket/AddAclBucketHandler.java

```diff
@@ -91,7 +91,11 @@ public Void call() throws Exception {
     boolean result = client.getObjectStore().addAcl(obj,
         OzoneAcl.parseAcl(acl));
 
-    System.out.printf("%s%n", "Acl added successfully: " + result);
+    String message = result
+        ? ("Acl added successfully: ") + result
```

Comment:
Can we change this message if true: Acl added successfully, in false, Acl already exists.
I think in false case printing Acl added successfully: false Acl already exists which is not great in my suggestion.

Reply:
Yes, I also think this message(```true``` : Acl added successfully.  ```false```: Acl already exists.)
would be more readable for developer and user.